src/views/corporate-training/hooks/useProgramDataFormatter.ts

```diff
@@ -7,16 +7,19 @@ interface FormattedData extends AdditionalData {
   slugDB: string
   categoryUrl: string
   ctDescription: string
+  categoryTitle?: string
 }
 
 export const useProgramDataFormatter = ({
   programData,
   isJobConnector = false,
-  selectedMethod
+  selectedMethod,
+  multipleMethod = true
```

Comment:
```suggestion
  multipleMethod = true
```
nama props multipleMethod disini rada kurang tepat gue rasa, karena tujuannya buat render [0] additionalData kan?

Reply:
`ignoreMutipleMethod` menurut gue lebih tepat, sambil dijelasin gunanya apa pake notes aja

Reply:
hmmm bener mas, cuman gw tadi rada bingung aja namingnya wkwkwk. Ada suggestion ga mas? selectFirstAdditionalData oke gak ya?

Reply:
ehh komen yang ignoreMultipleMethod tadi ga muncul di gw wkwk. Okehh boleh tuh mass